### PR TITLE
Setup transifex for version 7.0

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

**Comment by laetitia-gangloff** [comment]
:+1:

**Comment by lmignon** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -10,19 +10,24 @@ python:
...
-  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
 
 script:
-  - travis_run_flake8
   - travis_run_tests
 
 after_success:
```
Comment (pedrobaeza) [inline .travis.yml:33]: You can take the occasion to change this to travis_after_tests_success

Comment (guewen) [inline .travis.yml:33]: Right!

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**